scripts/tool_runner/qiskit_docs_contributor_tools/__init__.py

```diff
@@ -47,3 +49,41 @@ def run_cmd(name: str, cmd: list[str], *, progress: str | None) -> None:
 
     # This ends the program without adding a noisy stacktrace.
     raise SystemExit(1)
+
+
+def check_packages(verbose=False) -> None:
+    """Check the current system has the required packages"""
+    checks = [
+        (
+            "node",
+            (
+                "If you expect to use JavaScript in other projects, consider "
+                "using NVM (https://github.com/nvm-sh/nvm). Otherwise, consider "
+                "using Homebrew (https://formulae.brew.sh/formula/node) or "
+                "installing Node.js directly (https://nodejs.org/en)."
```

Comment:
The nodejs website automatically sets the locale based on the previous user choice. As in our app
```suggestion
                "installing Node.js directly (https://nodejs.org)."
```